### ISSUE TITLE
Instead of "Display Name", the community filter displays "Name" (URL)

### ISSUE THREAD

**Issue by loid345** [krawieck/lemmur#246]
The "name" is actually a url address and does not allow writing in Cyrillic and other letters other than Latin. If the platform is in Cyrillic, it is very difficult to understand the names of the community. I suggest replacing the filter of communities not by "Name", but by "Display Name".

**Comment by shilangyu** [comment]
Could you give an example? Preferably a specific community, I'll look into it

**Comment by krawieck** [comment]
i agree, tho i wouldn't do it globally, but instead add an option to toggle it. cuz i certainly prefer seeing the url name
